templates/back-office/crubside-arrive-content.hypr

```diff
@@ -0,0 +1,42 @@
+<div class="curbside-page-content">
+    <div>
+        <p>{{labels.curbsideSeeYouSoon}}</p>
+        <p>{{ labels.curbsideShipmentText }} </p>
+    </div>
+
+    <div class="curbside-spacing">
+        <button class="mz-delivery-button mz-button mz-button-large" onclick="location.href = 'http://{{domainName}}/anonymous-notification/curbsideInfo/{{model.shipmentNumber}}/{{model.orderId}}';"> {{ labels.CurbSidebtnText }}  </button>
```

Comment:
Don't hard code protocol. Use '//' instead.

Reply:
If this wasn't changed let's do it.